argus/util.py

```diff
@@ -392,6 +394,14 @@ def get_namedtuple(name, members, values):
     return nt_class(*values)
 
 
+def get_public_keys():
+    return get_resource("public_keys").splitlines()
+
+
+def get_certificate():
+    return get_resource("windows/certificate")
```

Comment:
This is Windows specific and you're using it for a mocked service which is platform-agnostic. I would prefer in this case to have these certificates in a common location.